samples/bluetooth/peripheral_nucleo_wb55rg/README.rst

```diff
@@ -0,0 +1,31 @@
+.. _bluetooth-peripheral_nucleo_wb55rg:
+
+Bluetooth: STM32WB55 BLE peripheral demo
+########################################
+
+Overview
+********
+
+Application demonstrating BLE peripheral on `ST Nucleo WB55RG`_
```

Comment:
Let's make this a sentence, something like:
```
This application demonstrates BLE peripherals on an `ST Nucleo WB55RG`_
by exposing a battery service and vendor-specific GATT services.
```

Reply:
done